concent_api/api-integration-multi-threads-test.py

```diff
@@ -0,0 +1,214 @@
+
```

Comment:
Delete this blank line

Reply:
done

Reply:
Actually this should stay, because it separates Python stdlib imports from 3rd party imports.

Reply:
this line was at the beginning of file. But I found place with this separate and add line

Reply:
That is strange. When I have refreshed page it showed me different line from this which I have marked.

Reply:
That's because of how Github stores code comments. It does not simply store file name and line number. It stores the whole patch you see above. That's also how it can detect that a comment is "outdated" - it sees that the diff it has is no longer present in the code. That way it can still find the right place in the code even if you move it without fixing the problem.

The downside of this approach is that when the diff is trivial, like above (the patch consists of a single empty line being added), the diff might match some other place in the code. So when @pawelkisielewicz has removed the empty line Github simply found another empty line and attached the comment there :)